content/administration/on_premise/deploy.rst

```diff
@@ -600,6 +600,141 @@ It should be stored securely, and should be generated randomly e.g.
 
+Reset the master password
+-------------------------
+
+There may be instances where the master password is misplaced or compromised and needs to be reset.
+The following process is for system administrators of an Odoo on-premise database to manually reset
+and re-encrypt the master password.
+
+.. seealso::
+   For more information about changing an Odoo.com account password, see this documentation:
+   :ref:`odoocom/change_password`.
+
+When creating a new on-premise database, a random master password is generated. Odoo recommends
+using this password to secure the database. This password is implemented by default so there is a
+secure master password for any Odoo on-premise deployment.
+
+.. warning::
+   When creating an Odoo on-premise database the installation is accessible to anyone on the
+   internet until this password is set to secure the database.
+
+The master password is specified in the Odoo configuration file (`odoo.conf` or `odoorc` (hidden
+file)). The Odoo master password is needed to modify, create, or delete a database through the
+graphical user interface (GUI).
+
+Locate configuration file
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First, :guilabel:`Open` the Odoo configuration file (`odoo.conf` or `odoorc` (hidden file)).
+
+.. tabs::
+
+   .. tab:: Windows
+
+      The configuration file is located at: `c:\\ProgramFiles\\Odoo{VERSION}\\server\\odoo.conf`
+
+   .. tab:: Linux
+
+      Depending on how Odoo is installed on the Linux machine, the configuration file is located in
+      one of two different places:
+
+      - Package installation: `/etc/odoo.conf`
+      - Source installation: `~/.odoorc`
+
+Change old password
+~~~~~~~~~~~~~~~~~~~
+
+Then, modify the old password in the configuration file to a temporary password.
+
+.. tabs::
+
+   .. tab:: Graphical user interface
+
+      After locating the configuration file, open it using a (:abbr:`GUI (graphical user
+      interface)`). This can be achieved by simply double clicking on the file and the device should
+      have a default :abbr:`GUI (graphical user interface)` to open the file with.
+
+      Next, modify the line to the master password `admin_passwd = $pbkdf2-sha…` to `admin_passwd =
+      newpassword1234`. This password can be anything as long as it is saved temporarily. Modify all
+      characters after the `=`.
+
+      .. example::
+         The line appears like this:
+         `admin_passwd =
+         $pbkdf2-sh39dji295.59mptrfW.9z6HkA$w9j9AMVmKAP17OosCqDxDv2hjsvzlLpF8Rra8I7p/b573hji540mk/.3ek0lg%kvkol6k983mkf/40fjki79m`
+
+         The modified line appears like this: `admin_passwd = newpassword1234`
+
+   .. tab:: Command-line interface
+
+      Modify the line to the master password using the following Unix command.
+
+      Connect to the Odoo server's terminal via Secure Shell (SSH) protocol and edit the
+      configuration file. To modify the configuration file, enter the following command:
+      :command:`sudo nano /etc/odoo.conf`
```

Comment:
```suggestion
      :command:`sudo nano /etc/odoo.conf`
```
is that space between "nano" and the first "/" intended? It seems like this has be to entered precisely in order to work properly. Just tossing it out there as a "double-check" suggestion.